tools/wptrunner/wptrunner/testrunner.py

```diff
@@ -978,12 +980,12 @@ def teardown(self):
         self.remote_queue = None
         self.recording.pause()
 
-    def ensure_runner_stopped(self):
-        self.logger.debug("ensure_runner_stopped")
+    def _ensure_runner_stopped(self):
         if self.test_runner_proc is None:
             return
 
-        self.browser.stop(force=True)
```

Comment:
I don't know in what scenario `force=False` will be useful. And will this be the majority of the case in future if we remove this line?

Maybe we should always do a forced kill instead. If that is agreed, we can do a cleanup in a separate PR.

Reply:
Filed #48066 for tracking cleanup/next steps. Looks like for the common case of restart due to crash/timeout, [`force=False` could become very common][1] (and potentially leaky), so I put back `force=True` for now.

[1]: https://github.com/web-platform-tests/wpt/blob/9b6507e295ccf5ea01506847706c899050fa6151/tools/wptrunner/wptrunner/testrunner.py#L866